fractal.js

```diff
@@ -0,0 +1,51 @@
+(function () {
+  // Create Canvas
+  var myCanvas = document.createElement("canvas");
+  myCanvas.width = document.documentElement.clientWidth;
+  myCanvas.height = document.documentElement.clientHeight;
+  // myCanvas.width = 1200;
+  // myCanvas.height = 300;
+
+  document.body.appendChild(myCanvas);
+  var ctx = myCanvas.getContext("2d");
+
```

Comment:
Overrall from your vid i can see the load time is a bit high so maybe consider optimizing these functions for runtime